lib/components/SelectionCard/SelectionCard.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { useMediaQueries } from '../../main';
+import { Tooltip } from '../Tooltip/Tooltip';
+import { TooltipContent } from '../Tooltip/TooltipContent';
+import { TooltipTrigger } from '../Tooltip/TooltipTrigger';
+
+export interface SelectionCardProps {
+  /** Card label */
+  label: string;
+  /** Icon content */
+  icon?: React.ReactNode;
+  /** Show the selection checkmark */
+  selected?: boolean;
+  /** Callback when the card is clicked */
+  onClick?: () => void;
+  /** Callback when the card is hovered/exited or focused/blurred */
+  setHovered?: (visible: boolean) => void;
+  /** Tooltip content that's displayed when the info button is pressed (mobile only) */
+  tooltipContent?: React.ReactNode;
+  /** Aria label for the info button */
+  infoAriaLabel?: string;
```

Comment:
I think that `infoAriaLabel` prop should be required as without it screenreader just seems to tell `button` when getting focus and it is not enough informative.

Reply:
You're right, I'll set it as required